src/events/crawler/scrapers/AUS/QLD/index.js

```diff
@@ -16,18 +24,35 @@ const scraper = {
   state: 'Queensland',
   type: 'paragraph',
   url: 'https://www.health.qld.gov.au/news-events/doh-media-releases',
-  async scraper() {
-    const $ = await fetch.page(this.url);
-    const anchors = $('#content h3:first-of-type > a');
-    const currentArticleUrl = anchors[0].attribs.href;
-    const $currentArticlePage = await fetch.page(currentArticleUrl);
-    const paragraph = $currentArticlePage('#content h2:first-of-type + p').text();
-    const paragraphWithoutNbsp = paragraph.replace(/\u00a0/g, ' ');
-    const { casesString } = paragraphWithoutNbsp.match(/state total to (?<casesString>\d+)./).groups;
-    return {
-      state: scraper.state,
-      cases: parse.number(casesString)
-    };
+  scraper: {
+    '0': async function() {
+      const $ = await getCurrentArticlePage(this.url);
+      const paragraph = $('#content h2:first-of-type + p').text();
+      const paragraphWithoutNbsp = paragraph.replace(/\u00a0/g, ' ');
+      const { casesString } = paragraphWithoutNbsp.match(/state total to (?<casesString>\d+)./).groups;
+      return {
+        state: scraper.state,
+        cases: parse.number(casesString)
+      };
+    },
+    '2020-3-24': async function() {
+      const HHSs = []; // See https://www.health.qld.gov.au/maps
+      const $ = await getCurrentArticlePage(this.url);
+      const $table = $('#content table');
+      const $trs = $table.find('tbody > tr:not(:last-child)');
+      $trs.each((index, tr) => {
+        const $tr = $(tr);
+        const cases = parse.number($tr.find('td:last-child').text());
+        const county = parse.string($tr.find('td:first-child').text());
+        HHSs.push({
+          county,
```

Comment:
Hmm, the previous scraper returned states, now this one has counties? Trying to wrap my head around the geography.

Reply:
Previous scraper was a state. Now it has subdivided areas called HHS. Can see the map in the comment.
Potentially I’d marked it wrong either before or after, but the idea is going from one state (Queensland) to a dozen or so sub-areas (which I figure aligns to USA counties).

Annoyingly one of the sub areas is “overseas” so not sure how to handle that either.